src/cy_serial_bridge/usb_constants.py

```diff
@@ -59,7 +59,7 @@ class CyType(IntEnum):
         6  # Used to indicate a device which is in CDC UART mode (which will automatically work using an OS driver)
     )
     UART_PHDC = 7  # Used to indicate a device which is in PHDC (Personal Healthcare Device Class) UART mode
-
+    CDC_DATA = 8  # Used to indicate the CDC data interface (which is a separate interface from the CDC UART interface)
```

Comment:
Random question, do you have a source for what this interface actually does?  I don't actually know, and I think I wasn't able to find out easily back when I was looking into this.

Reply:
No, I was curious myself... There is a CDC defintion for SPI devices also, maybe its related to that?